pkg/proxy/ipvs/proxier.go

```diff
@@ -997,14 +997,14 @@ func (proxier *Proxier) syncProxyRules() {
 	activeIPVSServices := map[string]bool{}
 	// currentIPVSServices represent IPVS services listed from the system
 	currentIPVSServices := make(map[string]*utilipvs.VirtualServer)
-	// activeBindAddrs represents ip address successfully bind to defaultDummyDevice in this round of sync
-	activeBindAddrs := map[string]bool{}
-	// Addresses assigned to the dummy interface
+	// activeBindAddrs Address we want on the defaultDummyDevice after this round of sync
```

Comment:
need to keep the word "represents" or else put some punctuation in there
(likewise below)

Reply:
fixed